.releaserc

```diff
@@ -1,9 +1,8 @@
 {
   "branches": [
-    "master",
-    "next",
-    { "name": "alpha", "prerelease": true },
-    { "name": "beta", "prerelease": true }
+    { "name": "release-22.x", "range": "22.x", "channel": "22.x" },
+    { "name": "release-23.x", "range": "23.x" },
```

Comment:
if I understand the `channel` docs (https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#channel) correctly this should make it so the `release-23.x` branch publishes to the `@latest` tag on npm